scripts/generateAssetData/generateRelatedAssetIndex/mapping.ts

```diff
@@ -20,3 +26,24 @@ export const zerionImplementationToMaybeAssetId = (
   })()
   return toAssetId({ chainId, assetNamespace, assetReference })
 }
+
+export const coingeckoPlatformDetailsToMaybeAssetId = (
+  platformDetails: [string, string],
+): AssetId | undefined => {
+  const [platform, contractAddress] = platformDetails
+  const chainId = adapters.coingeckoAssetPlatformToChainId(
+    platform as adapters.CoingeckoAssetPlatform,
```

Comment:
suggestion: we could avoid this cast by typing all the way from the axios request to this utils, the lift doesn't look so big, consumed at 3 small places

Reply:
We can't do this as `Object.entries()` will loosen type inference e.g after adding types at response-level:
<img width="859" alt="image" src="https://github.com/user-attachments/assets/437e4b3d-9ef0-4917-82ae-1c37721d1574">
but: 
<img width="1438" alt="image" src="https://github.com/user-attachments/assets/21eca679-d71b-4108-acda-7b571ee9ea7c">